quickwit-common/src/lib.rs

```diff
@@ -30,6 +31,10 @@ use tracing::{error, info};
 /// Filenames used for hotcache files.
 pub const HOTCACHE_FILENAME: &str = "hotcache";
 
+pub fn get_hotcache_path() -> &'static Path {
```

Comment:
This function is not used I think. You can put `HOTCACHE_FILENAME` directly in the bundle directory module as it's only used here, currently you don't use the variable `PathBuf::from("hotcache".to_string()),`

Reply:
they are  both not used anymore :)